### PR TITLE
add changelog for 0.7.0 release

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- add changelog for 0.7.0 release

#### Ticket Link
n/a

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by k4leung4 — DISMISSED** [review]
@haydentherapper @priyawadhwa Any other PRs we should be waiting on for this release?

This has the removal of the timestamping authority and the shard downtime improvement.

**Review by priyawadhwa — DISMISSED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,64 @@
+# v0.7.0
+
+## Enhancements
+
+* Remove timestamping authority (#813)
```
Comment (Hayden-IO) [inline CHANGELOG.md:9]: Can we call this one out loudly, since it's a breaking change to the API?

**Review by Hayden-IO — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cpanato) [reply CHANGELOG.md:9]: done PTAL, thanks for the review @haydentherapper

**Review by k4leung4 — APPROVED** [review]

**Review by Hayden-IO — APPROVED** [review]

**Review by dlorenc — APPROVED** [review]

**Pull request merged**